### DISCUSSION TITLE
Background color Input addOnAfter

### DISCUSSION THREAD

**Discussion by rustam-as** [ant-design/ant-design#41031] [Q&A]
### Reproduction link

[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/cocky-euler-oulccb)

### Steps to reproduce

I need to change addOnAfter background color.

### What is expected?

I tried several ways to modify background color. But i guess antd did not consider addOnAfter could need different bg color

### What is actually happening?

when styling with inline css directly the icon it doesnt cover all background .

| Environment | Info |
| --- | --- |
| antd | 5.2.3 |
| React | react@18.2.0 |
| System | Windows 11 pro 64bit (10.0 Build 22000) |
| Browser | Google Chrome Version 110.0.5481.77 (Official Build) (64-bit) |

**Comment by Wxh16144** [comment]
Have you tried to override the `.ant-input-group-addon` class name by means of a css module? It should work.

<img width="707" alt="image" src="https://user-images.githubusercontent.com/32004925/222452789-5b1753ac-aed2-4712-85f5-16a1474393ef.png">

**Comment by rustam-as** [comment]
> Have you tried to override the `.ant-input-group-addon` class name by means of a css module? It should work.

no I didnt, I hopefully thought maybe there is some better way. thank you for reminding
